packages/suite/src/components/suite/modals/index.tsx

```diff
@@ -46,23 +45,27 @@ import { RawRenderer } from '../Modal/RawRenderer';
 import type { AcquiredDevice } from '@suite-types';
 
 const useSharedProps = () => {
-    const { isGuideOpen } = useGuide();
+    const { isGuideOpen, isGuideOnTop } = useGuide();
+
     const props = useSelector(state => ({
         modal: state.modal,
         device: state.suite.device,
         devices: state.devices,
         router: state.router,
     }));
+
     const actions = useActions({
         onCancel: allModalActions.onCancel,
         onPinCancel: allModalActions.onPinCancel,
         onReceiveConfirmation: allModalActions.onReceiveConfirmation,
         goto: routerActions.goto,
     });
+
     return {
         ...props,
         ...actions,
         isGuideOpen,
+        isGuideOnTop,
```

Comment:
Are these 'guide' values still necessary in `useSharedProps`, given that the general `Modal` component obtains them itself?

Reply:
done